posthog-node/src/crypto.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference lib="dom" />
+
+import { getNodeCrypto, getWebCrypto } from './crypto-helpers'
+export async function hashSHA1(text: string): Promise<string> {
```

Comment:
Any chance we could cache the importing/crypto detection? This function runs pretty often. Maybe we could have a top-level await and abstract this away entirely?

Reply:
Good idea. I'll try that.